apis/projectcontour/v1/httpproxy.go

```diff
@@ -137,6 +139,13 @@ type HeaderMatchCondition struct {
 	// value.
 	// +optional
 	Regex string `json:"regex,omitempty"`
+
+	// TreatMissingAsEmpty specifies if the header match rule specified header
+	// does not exist, this header value will be treated as empty. Defaults to false.
+	// Unlike the underlying Envoy implementation this is **only** supported for
+	// negative matches (e.g. NotContains, NotExact).
+	// +optional
+	TreatMissingAsEmpty bool `json:"TreatMissingAsEmpty,omitempty"`
```

Comment:
```suggestion
	TreatMissingAsEmpty bool `json:"treatMissingAsEmpty,omitempty"`
```

Reply:
fixed in bdd8b20a58135f00906b651196b6240e56dba593